components/post_profile_picture/post_profile_picture.jsx

```diff
@@ -70,16 +78,21 @@ export default class PostProfilePicture extends React.PureComponent {
             user,
             isBot,
         } = this.props;
+
         const isSystemMessage = PostUtils.isSystemMessage(post);
         const fromWebhook = PostUtils.isFromWebhook(post);
+
         if (isSystemMessage && !compactDisplay && !fromWebhook && !isBot) {
             return <MattermostLogo className='icon'/>;
         }
-
         const fromAutoResponder = PostUtils.fromAutoResponder(post);
 
         const hasMention = !fromAutoResponder && !fromWebhook;
-        const src = this.getProfilePicSrcForPost(fromAutoResponder, fromWebhook);
+        const profileSrc = this.getProfilePictureURL();
+        const src = this.getPostIconURL(profileSrc, fromAutoResponder, fromWebhook);
+
+        // if it's relative we assume it's a link to an emoji
+        const isEmoji = src.startsWith('/') && src !== Constants.DEFAULT_WEBHOOK_LOGO;
```

Comment:
Same comment as on mobile. I don't think we can safely make this assumption with plugins being able to host their own profile pictures